status-im/status-react
# fix: avatar alignment

Fixes https://github.com/status-im/status-mobile/issues/18827

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
Handles the limit event when creating a new group chat
- Fixes avatar/username alignment
- Adds animation for avatar/username when scrolling up ([ref](https://discord.com/channels/624634427930312714/928625369542713396/1213089049103835259))

#### Platforms


- Android
- iOS

##### Functional

- 1-1 chats

### Steps to test


- Open Status
- Go to chat tab
- Start 1:1 chat with a contact
- When you have enough content more than the screen, scroll to the bottom
- Observe animation and alignment for avatar/username

status: ready 

![Avatar Alignment Fix](https://github.com/status-im/status-mobile/assets/45393944/32585f55-9f69-4192-98cf-1d10fc1979cf)
